src/application/action-builder.js

```diff
@@ -47,12 +49,22 @@ function _dispatchServiceResponse({ node, type, status, callerId }, json) {
         status: newStatus,
         callerId
     });
+
+    // Update information similar to store::afterChange
+    return Promise.resolve({
+        properties: Object.keys(data),
```

Comment:
I feel like we also need to send data, or the content of the response here.
The whole purpose is to have it, and not to go get it from store.